src/content/docs/cheerpj3/11-guides/Implementing-Java-native-methods-in-JavaScript.md

```diff
@@ -11,20 +11,27 @@ As an example, consider the following Java class:
 package com.example;
 
 public class TestClass {
-  public native void alert(String str);
+
+	public static native void nativeAlert(String url);
```

Comment:
Why `url`?

Reply:
I'm actually unsure, will change it to str